MvvmCross-Plugins/Email/MvvmCross.Plugins.Email.Droid/MvxComposeEmailTask.cs

```diff
@@ -71,31 +75,63 @@ public void ComposeEmail(string to, string cc = null, string subject = null, str
                 var uris = new List<IParcelable>();
 
                 DoOnActivity(activity => {
-                    foreach (var file in attachments)
-                    {
-                        File localfile;
-                        using (var localFileStream = activity.OpenFileOutput(
-                            file.FileName, FileCreationMode.WorldReadable))
-                        {
-                            localfile = activity.GetFileStreamPath(file.FileName);
-                            file.Content.CopyTo(localFileStream);
-                        }
-                        localfile.SetReadable(true, false);
-                        uris.Add(Uri.FromFile(localfile));
-                        localfile.DeleteOnExit(); // Schedule to delete file when VM quits.
-                    }
+                    filesToDelete = new List<Java.IO.File>();
+
+					foreach (var file in attachments)
```

Comment:
This is fixed in the next commit of this pull request: https://github.com/MvvmCross/MvvmCross/pull/1648/commits/937eee05065c4ae296163145ad626ec5afa9787e